cmd/cli/plugin/package/main.go

```diff
@@ -33,6 +35,13 @@ func main() {
 		log.Fatal(err)
 	}
 
+	if config.IsFeatureActivated(config.FeatureFlagPackagePluginKctrlCommandTree) {
+		if err := kctrl.Invoke(p); err != nil {
+			os.Exit(1)
+		}
+		return
```

Comment:
Could I ask you please test this manually by disabling this feature flag and making sure legacy package plugin commands are working properly?

Reply:
Yep, I have tested with both the feature flag enabled and disabled. I have also not updated the existing package plugin integration tests so they are also passing for the existing behaviour of package plugin (i.e. the feature flag being disabled).